test/AutoRest.TestServer.Tests/Mgmt/TestProjects/TestProjectTests.cs

```diff
@@ -53,42 +52,74 @@ public void ValidateMethodExists(string methodName)
             }
         }
 
-        private IEnumerable<Type> FindAllOperations()
+        [Test]
+        public void ValidateResourceGroupExtensions()
         {
+            if (_projectName.Equals(""))
```

Comment:
let this fail, this should never happen and we want to know

Reply:
Test failed
![image](https://user-images.githubusercontent.com/30464227/115508327-c1d6fa00-a2af-11eb-99e0-c4d08e875c20.png)